iroha/src/sumeragi/mod.rs

```diff
@@ -490,6 +493,18 @@ impl<G: GenesisNetworkTrait, W: WorldTrait> Sumeragi<G, W> {
         Ok(())
     }
 
+    async fn gossip_msg_to(
```

Comment:
We already have `send_to_all`

Reply:
This is broadcasting to some specific number of peers